charts/hail-search/values.yaml

```diff
@@ -74,6 +84,14 @@ affinity: |-
 nodeSelector: 
   {}
 
+sourceCachedReferenceDatasetQueriesDir: 'gs://seqr-reference-data/v03'
+cachedReferenceDatasetQueries:
+  GRCh38/SNV_INDEL:
```

Comment:
I fixed up the reference dataset paths earlier this week so that both the cached queries and the combined collections have this structure.